packages/mock-block-dock/src/mock-block-dock-ui.tsx

```diff
@@ -26,17 +25,89 @@ export const MockBlockDockUi: FunctionComponent<{ children: ReactNode }> = ({
     <DebugView>{childrenWithBorder}</DebugView>
   ) : (
     <Box>
-      <Box className={styles["mbd-debug-mode-toggle-header"]}>
+      <div className="mbd-debug-mode-toggle-header">
         <button
-          className={styles["mbd-debug-mode-toggle"]}
+          className="mbd-debug-mode-toggle"
           type="button"
           onClick={() => setDebugMode(true)}
         >
           Preview Mode
           <OnSwitch />
         </button>
-      </Box>
+      </div>
       {childrenWithBorder}
+      <style jsx global>{`
+        @font-face {
+          font-family: "Inter";
+          font-weight: 300;
+          src: url("https://cdn-us1.hash.ai/assets/fonts/Inter-Italic.woff2")
+            format("woff2");
+        }
+
+        @font-face {
+          font-family: "Inter";
+          font-weight: 400;
+          src: url("https://cdn-us1.hash.ai/assets/fonts/Inter-Regular.woff2")
+            format("woff2");
+        }
+
+        @font-face {
+          font-family: "Inter";
+          font-weight: 500;
+          src: url("https://cdn-us1.hash.ai/assets/fonts/Inter-Medium.woff2")
+            format("woff2");
+        }
+
+        @font-face {
+          font-family: "Mono";
+          font-weight: 400;
+          src: url("https://cdn-us1.hash.ai/assets/fonts/jet-brains-mono-regular.woff2")
+            format("woff2");
+        }
+
+        body {
+          margin: 0;
+        }
+      `}</style>
+      <style jsx>
+        {`
+          .mbd-debug-mode-toggle-header {
+            display: flex;
+            justify-content: flex-end;
+            align-items: center;
+            padding-left: 24px;
+            padding-right: 24px;
+            border-bottom: 1px solid rgba(0, 0, 0, 0.12);
+            height: 50px;
+          }
+
+          .mbd-debug-mode-toggle {
+            font-family: Inter, "Helvetica", "Arial", sans-serif;
+            z-index: 1000;
+            border: 1px solid #dde7f0;
+            color: #37434e;
+            background: transparent;
+            font-size: 0.8125rem;
+            line-height: 1.75;
+            padding: 3px 9px;
+            border-radius: 4px;
+            cursor: pointer;
+            align-content: baseline;
+            display: flex;
+            align-items: center;
+          }
+
+          :global(.mbd-debug-mode-toggle > svg) {
+            height: 20px;
+            width: 40px;
+            margin-left: 10px;
+          }
```

Comment:
How about passing `className` to `OnSwitch` instead? Seems like `:global` is avoidable (unless I miss something)

Reply:
Yep fair enough - used `sx` instead, in https://github.com/blockprotocol/blockprotocol/pull/650/commits/bc95c5ba8569ec3252d38b488ad3fe256e12f062